resources/language/resource.language.es_es/strings.po

```diff
@@ -158,7 +160,7 @@ msgstr "ESN manual"
 
 msgctxt "#30035"
 msgid "InputStream Adaptive settings..."
-msgstr "Ajustes de InputStream Adaptive..."
+msgstr "Configurar InputStream Adaptive..."
 
```

Comment:
DON'T AGREE for consistency with the rest of the translation. Use my previous translation (but both are perfectly valid).

Reply:
Resolved @CastagnaIT has changed it